src/hooks/comments/useComments.js

```diff
@@ -0,0 +1,7 @@
+import { useSelector, shallowEqual } from 'react-redux';
+
+export function useComments() {
+  const request = useSelector((state) => state.comments.update, shallowEqual);
+
+  return { request };
```

Comment:
```suggestion
  return request;
```
return just a request object not `{request: request}`